jeos-firstboot-enroll

```diff
@@ -206,3 +278,19 @@ enroll_post() {
 		run sdbootutil add-all-kernels --no-reuse-initrd
 	fi
 }
+
+enroll_jeos_config() {
+	d --insecure --passwordbox  $"Enter encryption password" 0 0
+	[ -n "$result" ] || return 0
+	echo -n "$result" | keyctl padd user cryptenroll @u
+
+	if ! detect_luks2; then
```

Comment:
again, I would reverse the check: if there is no luks device -- nothing to do and then ask for the recovery password

Reply:
yes. the check works differently now